gss-client/gss-github-client/src/main/java/com/devoops/client/GitHubWebhookRequest.java

```diff
@@ -0,0 +1,31 @@
+package com.devoops.client;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public record GitHubWebhookRequest(
+    String name,
+    boolean active,
+    List<String> events,
+    GitHubWebhookConfig config
+) {
+    public record GitHubWebhookConfig(
+        String url,
+        @JsonProperty("content_type")
+        String contentType
+    ) {
+    }
+
+    public static GitHubWebhookRequest ofPullRequestEvent(String webhookUrl) {
+        return new GitHubWebhookRequest(
+            "web",
+            true,
+            List.of("pull_request"),
+            new GitHubWebhookConfig(
+                webhookUrl,
+                "json"
```

Comment:
[매우매우 사소한 제안]

```suggestion
           MediaType.APPLICATION_JSON.getSubtype()
```

하드코딩보다 미디어타입 enum으로 활용해도 좋을 것 같아서 제안드려 보아요 😄

Reply:
기존에 정의해둔거 사용하는는게 실수 줄이는 측면에서 좋은 것 같습니다.

[5da284d](https://github.com/mash-up-kr/GSS-Server/pull/11/commits/5da284d3b447144a53928bc05747c82e50e456f3) 에서 반영완료🙇‍♀️